umm-spec-lib/src/cmr/umm_spec/date_util.clj

```diff
@@ -81,11 +81,25 @@
   [date]
   (let [date (f/parse date)]
     (try
-      (t-core/equal? date
+      (time-core/equal? date
                      date)
       (catch
         Exception e nil))))
 
+(defn is-past?
+  "Parse date and return true if date is in the past, false if not, nil if date
+  is nil"
+  [date]
+  (when date
+    (time-core/before? date (time-keeper/now))))
+
+(defn is-future?
```

Comment:
Nitpick: this name might be confusing, because `futures` are a thing in clojure

https://clojuredocs.org/clojure.core/future